src/openrct2/rct2/S6Exporter.cpp

```diff
@@ -1180,7 +1180,6 @@ template<> void S6Exporter::ExportEntity(RCT2SpriteVehicle* dst, const Vehicle*
     dst->dodgems_collision_direction = src->dodgems_collision_direction;
     dst->animation_frame = src->animation_frame;
     dst->animationState = src->animationState;
-    dst->var_CA = src->var_CA;
```

Comment:
This needs to be done correctly, you have to split them from animationState back into the two.

Reply:
it can be done by just changing the definition of RCT2SpriteVehicle to make animationState uint32_t

Reply:
I didn't realize, yeah that should be fine.

Reply:
Turns out there was some reorg required.